### PR TITLE
SDK-3037 Update Segment Android SDK to v5.0.0

### PR DESCRIPTION
**Pull request by darshanclevertap**
- Updates Android SDK to v5.0.0
- Updates Segment analytics-android to v4.11.3

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -55,14 +55,14 @@ group = publishedGroupId
 
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
     buildToolsVersion '30.0.3'
```
Comment (piyush-kukadiya) [inline build.gradle]: @darshanclevertap buildToolsVersion need to change to 33.0.0

Comment (darshanclevertap) [reply build.gradle]: @piyush-kukadiya Thanks for pointing. Will do it

**Review by piyush-kukadiya — APPROVED** [review]
LGTM

**Review by shivamsharma2710 — APPROVED** [review]
LGTM

**Pull request merged**